### PR TITLE
BaSD-MS-02: Administrators Components

### PR DESCRIPTION
**Pull request by Pablo-Balbo**
## Description:
Creation of new Administrator Components to manage the Admin CRUD.
### Steps to reproduce:
1. Run `npm run start` on both front and back-end apps.
2. Search for **admin** option on nav bar and click on it.
3. Verify that the Administrators lists is rendered correctly.
4. Check **delete**, **edit** and **add** features are working properly.
### Reviewers:
- @estefrare
- @PabloMBernal

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,9 +1,95 @@
...
+    fetch(`${process.env.REACT_APP_API}/admins/${selectedAdmin}`, { method: 'DELETE' })
+      .then((response) => {
+        if (response.status !== 200) {
+          return response.json().then(({ message }) => {
+            throw new Error(message);
+          });
+        }
+        return fetch(`${process.env.REACT_APP_API}/admins`)
+          .then((response) => {
+            if (response.status !== 200) {
```
Comment (nicolasblasco) [inline src/Components/Admins/index.js:37]: @Pablo-Balbo the http status code should be 204

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,9 +1,95 @@
...
+            <th>Username</th>
+          </tr>
+        </thead>
+        <tbody>
+          {admins.map((admin) => (
+            <tr key={admin._id}>
+              <td>{admin.name}</td>
+              <td>{admin.username}</td>
+              <td>
+                <button>Edit</button>
```
Comment (nicolasblasco) [inline src/Components/Admins/index.js:83]: @Pablo-Balbo we should add a function to open the form when we click the edit button

File: src/Components/Admins/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            });
+          }
+          return response.json();
+        })
+        .then((response) => {
+          console.log(response);
+          // TODO
+          // setFullNameValue();
+          // setUsernameValue();
+          // setPasswordValue();
```
Comment (nicolasblasco) [inline src/Components/Admins/Form/index.js:30]: @Pablo-Balbo we should save the fields values in a state

File: src/Components/Admins/index.js (hunk 2 above)
Comment (Pablo-Balbo) [reply src/Components/Admins/index.js]: Done!

File: src/Components/Admins/Form/index.js (hunk 3 above)
Comment (Pablo-Balbo) [reply src/Components/Admins/Form/index.js]: In progress. I can't show the correct data of the admin that i want to edit.

Comment (Pablo-Balbo) [reply src/Components/Admins/Form/index.js]: Done!

**Review by estefrare — APPROVED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**